### PR TITLE
More complete build for Travis CI

### PR DESCRIPTION
**Pull request by msgilligan**
... so more build errors are caught by CI.

Gradle `build` rather than `check` in Travis CI. Also remove the exclusion of `:wallettemplate:check`.

This also addresses Issue #1707

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jdk:
 install:
   - sudo apt-get install -y protobuf-compiler
 script:
-  - gradle clean check -x :wallettemplate:check
+  - gradle clean build
```
Comment (schildbach) [inline .travis.yml]: This won't work -- we need to exclude wallettemplate from the build since there is no reliable way to declare the JavaFX dependency.

Comment (schildbach) [reply .travis.yml]: I suggest #2760.

**Comment by schildbach** [comment]
Fixed via 3974b38bad78c703a358650f78c9e0ba6ce6b82b

File: .travis.yml (hunk 1 above)
Comment (msgilligan) [reply .travis.yml]: Yeah sorry, my local build was using Oracle JDK8.

> there is no reliable way to declare the JavaFX dependency

I might be interested in trying to find a solution for that in the future. There's got to be a way.

> I suggest #2760.

What are you referring to by #2760?

Comment (schildbach) [reply .travis.yml]: No idea where I got this number from. I actually meant  #1711.

**Pull request closed without merging**